docs/prow/production-cluster-configuration.md

```diff
@@ -72,6 +73,7 @@ Use the following tools and configuration:
    - Deploy Prow components with the `a202e595a33ac92ab503f913f2d710efabd3de21`revision.
    - Deploy the Cert Manager.
    - Deploy secure Ingress.
+   - Deploy the [Prow Addons Controller Manager](../../development/prow-addons-ctrl-manager/README.md).
```

Comment:
will be visible after merging this PR: https://github.com/kyma-project/test-infra/pull/592